apps/src/code-studio/pd/workshop_dashboard/components/workshop_form.jsx

```diff
@@ -1117,6 +1148,71 @@ export class WorkshopForm extends React.Component {
               )}
             </Col>
           </Row>
+          <Row>
```

Comment:
just curious if this should be inside the conditional?

Reply:
Good q! I just double checked the formatting without the 'build your own' selected and it looks fine. That said I can pull it inside in my next PR that pipes in the pl offerings! Just to get this one merged:)